src/chttpd/src/chttpd.erl

```diff
@@ -141,7 +141,12 @@ start_link(Name, Options) ->
     end.
 
 get_server_options(Module) ->
-    ServerOptsCfg = config:get(Module, "server_options", "[]"),
+    ServerOptsCfg =
+        case Module of
+            "chttpd" -> config:get(Module,
+                "server_options", "[{recbuf, undefined}]");
```

Comment:
Maybe we could `-define(...)` a macro at the top to make it more readable. `DEFAULT_SERVER_OPTIONS` ?

Reply:
Changed.